### ISSUE TITLE
Redis: Update to 6 and supply the default config

### ISSUE THREAD

**Issue by MeFisto94** [parkervcp/eggs#596]
Panel Version: 0.7.17
Daemon Version: 0.6.13
Service: redis
Modified: no

So this is both, a bug report and a "game request" basically, sorry if I picked wrong.

Expected Behavior:
- Another egg or an updated image to support Redis 6 (SSL and others)
- The container being updated (https://quay.io/repository/parkervcp/pterodactyl-images/manifest/sha256:b65d17f6078ecba47fd59522d16633a22fc4e13b348da3bac8fa8359258c0384?tab=vulnerabilities)
- The default config file being passed (https://redis.io/topics/config) ` is available in the self documented example redis.conf shipped into the Redis distribution.`
- Console input/output should work

Actual Behavior:
- Redis 5 without any config that one could edit, without modifying the containers
- The console output is screwed (the container doesn't detect the starting sentence and typed input is invisible and when pressing enter on a correct command it seems printed a few times, tell me if you can't reproduce that)

Steps to Reproduce: (Step by step what to do to cause the issue)
- Use the Redis Egg

**Comment by parkervcp** [comment]
I only just found out redis 6 was out.

I am working on getting containers to rebuild regularly.

I set all the ports and passwords in the start command not using the redis config. That would be why it's not there by default.

the console is a default redis-cli tool. If it's not parsing then it's the redis-cli that's being weird.

**Comment by MeFisto94** [comment]
> I set all the ports and passwords in the start command, not using the redis config. That would be why it's not there by default.
Sure, but you can configure so many things for redis, and as such having a config would be awesome. I wonder if the container contains these default/example configs, but https://hub.docker.com/_/redis/ definitely supports that.

> If it's not parsing then it's the redis-cli that's being weird.
That's true, but I wonder if we could help that with stuff like `TERM=dumb` or others, I guess you had such issues with other containers before?

**Comment by parkervcp** [comment]
So with some testing I can use the default redis config, btw it's huge and can't be edited in the panel. I can get it started just like redis 5.

The issue with the cli is still happening. I don't have a fix for it.

**Comment by parkervcp** [comment]
@MeFisto94 can you try that please?

**Comment by MeFisto94** [comment]
I gave that a try and it works, I was even able to fix the cli issue with the aformentioned `TERM=dumb`,
I've tested it by specifying this as a variable in the egg but for safety reasons I also prepended a `TERM=dumb` to `redis-cli`.
The Panel Size can be adjusted using `PTERODACTYL_FILES_MAX_EDIT_SIZE` it seems, though it's sad that it doesn't fix the default config.

I have two remaining issues:
 1. The Running state is not detected properly. Even with a dumb terminal I had it working only once. It's only a minor thing, but since it should be simple string matching, it might be solved somehow.
 2. The shutdown mechanism. Now that the terminal works, one could specify SHUTDOWN as the command (without the need of chaining another redis-cli, but I guess that is to gracefully shutdown when redis-cli quits and also because only using `SHUTDOWN` does not close the redis-cli session and I don't know how to chain them).

So overall it works like a charm, thank you :)
It might be worth including `TERM=dumb` before the redis-cli call, though :)
